src/components/controls/PasswordInput/README.md

```diff
@@ -0,0 +1,39 @@
+## Password Input
+
+```tsx
+import {PasswordInput} from '@gravity-ui/uikit';
+```
+
+Password Input display component
```

Comment:
```suggestion
`TextInput` for typing passwords and other sensitive information. It can be rendered with copy and reveal buttons for more convinient usage.
```